laws/src/main/scala/cats/laws/ApplicativeErrorLaws.scala

```diff
@@ -39,6 +39,12 @@ trait ApplicativeErrorLaws[F[_], E] extends ApplicativeLaws[F] {
 
   def attemptFromEitherConsistentWithPure[A](eab: Either[E, A]): IsEq[F[Either[E, A]]] =
     F.attempt(F.fromEither(eab)) <-> F.pure(eab)
+
+  def onErrorPure[A, B](a: A, f: E => F[B]): IsEq[F[A]] =
+    F.onError(F.pure(a)){case err => F.void(f(err))} <-> F.pure(a)
```

Comment:
Yes you're right. I hadn't realised I needed to add it to the MonadErrorTests as well (because of `parent`). Thank you! (I'll update the PR tonight with the change)